src/components/Dialogue/dialogue.ts

```diff
@@ -0,0 +1,36 @@
+import Block from 'core/Block';
+
+import './dialogue.scss';
+
+interface dialogueProps {
+  name?: string;
+  textYou?: string;
+  text?: string;
+  time?: string;
```

Comment:
```suggestion
  time?: Date;
```
Лучше сделать тут тип Date. Если хранить время как строку, то будет очень много проблем с парсингом этой строк (в разных регионах разный формат времени). Если нужно вывести время на экран, то хорошим решением будет написать функцию, например
```
const timeFormatter = (time: Date): string => {}
```
которая вернет нужную нам строку